test/e2e/common/configmap.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-var _ = ginkgo.Describe("[sig-node] ConfigMap", func() {
+var _ = SIGNodeDescribe("ConfigMap", func() {
```

Comment:
this change added the `[k8s.io]` tag to the beginning as well, correct? I don't know the history of this tag, is it expected?

Reply:
oh, I found that discussion that the tag will be removed. nvm.